app/models/product_criteria/basic.rb

```diff
@@ -67,7 +67,7 @@ def well_location
   end
 
   def plate_barcode
-    @well_or_metric.plate.try(:human_barcode) || 'Unknown'
+    @well_or_metric.labware.try(:human_barcode) || 'Unknown'
   end
```

Comment:
Plate and labware are actually different associations, and labware is the one we're eager loading. Would be much nicer if rails let you alias an association. (As it is we can alias getters, but last I checked we couldn't alias for things like eager loading)